frontend/src/components/gradebook/gradebook.tsx

```diff
@@ -106,96 +107,104 @@ const Gradebook = (): ReactElement => {
   };
 
   return (
-    <div className="gradebook">
-      <div className="grade-navigation">
-        <div className="search">
-          <label
-            htmlFor="grade-search"
-            className={activeSearch ? "active" : ""}
-          >
-            Search for Student...
-          </label>
-
-          <input
-            id="grade-search"
-            onFocus={focus}
-            onBlur={blur}
-            onChange={searchStudent}
-          />
+    <>
```

Comment:
does everyone know about this <></> thing except for me?????

Reply:
idk what that's for either

Reply:
It's a dummy parent so that you don't have a div you're using just to satisfy react.

Reply:
https://reactjs.org/docs/fragments.html#short-syntax